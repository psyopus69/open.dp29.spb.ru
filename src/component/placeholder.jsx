import React, {Fragment} from "react";
import {PropTypes} from "prop-types";

// Complete Counter with buttons & timer
const Placeholder = (props) => {
  const {image} = props;
  return <Fragment>
    Loading Screen<img alt='placeholder' src={image} className='placeholderMain' />
  </Fragment>;
};
Placeholder.propTypes = {
  image: PropTypes.string.isRequired,
};

export {
  Placeholder,
};
