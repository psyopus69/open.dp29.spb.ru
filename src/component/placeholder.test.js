import React from "react";
import renderer from "react-test-renderer";
import {Placeholder} from "./placeholder";

it(`placeholder with custom image`, () => {
  const Tree = renderer.create(<Placeholder image={`assets/img/placeholder_1.gif`}/>);
  expect(Tree).toMatchSnapshot();
});
