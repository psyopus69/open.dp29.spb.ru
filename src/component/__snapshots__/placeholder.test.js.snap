// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`placeholder with custom image 1`] = `
Array [
  "Loading Screen",
  <img
    alt="placeholder"
    className="placeholderMain"
    src="assets/img/placeholder_1.gif"
  />,
]
`;
